app/controllers/api/permit_applications_controller.rb

```diff
@@ -79,8 +94,8 @@ def submit
     signed = permit_application_params["submission_data"]["data"]["section-completion-key"]["signed"]
 
     # for submissions, we do not run the automated compliance as that should have already been complete
-    if signed &&
-         @permit_application.update(permit_application_params.merge(status: :submitted, signed_off_at: Time.current))
+    if signed && @permit_application.using_current_template_version
+      @permit_application.update(permit_application_params.merge(status: :submitted, signed_off_at: Time.current))
```

Comment:
I think there should be an `&` here Before the logic only sent notifications if the update was a success